File: base/src/main/java/com/austindroids/austinfeedsme/eventsfilter/EventFilterContract.java
package com.austindroids.austinfeedsme.eventsfilter;

import com.austindroids.austinfeedsme.data.Event;

import java.util.List;

/**
 * Created by darrankelinske on 8/4/16.
 */
public class EventFilterContract {

    interface View {
        void showEvents(List<Event> events);
    }

    interface Presenter {
        void loadEvents();
    }

}

File: base/src/main/java/com/austindroids/austinfeedsme/eventsfilter/EventFilterContract.kt
package com.austindroids.austinfeedsme.eventsfilter

import com.austindroids.austinfeedsme.data.Event

/**
 * Created by darrankelinske on 8/4/16.
 */
interface EventFilterContract {

    interface View {
        fun showEvents(events: List<Event>)
    }

    interface Presenter {
        fun loadEvents()
        fun dispose();
    }
}